File: public/docs/_examples/attribute-directives/ts/src/app/highlight.directive.1.ts
// #docregion
import {Directive, ElementRef, Renderer, Input} from 'angular2/core';

@Directive({
    selector: '[my-highlight]'
})

export class Highlight {
    constructor(el: ElementRef, renderer: Renderer) {
        //el.nativeElement.style.backgroundColor = 'yellow';
       renderer.setElementStyle(el, 'background-color', 'yellow');
    }
}
// #enddocregion

File: public/docs/_examples/attribute-directives/dart/lib/highlight_directive_1.dart
// #docregion
library attribute_directives.highlight_directive;

import 'package:angular2/core.dart';

@Directive(selector: '[myHighlight]')
class HighlightDirective {
  HighlightDirective(ElementRef element) {
    element.nativeElement.style.backgroundColor = 'yellow';
  }
}
